index.html

```diff
@@ -48,24 +56,44 @@
     </style>
   </head>
 
+  <!-- The <body> element defines the document's body, and is a container for all the visible contents that will be rendered by the browser.  -->
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <!-- The <h1> element defines a large heading. There are 6 pre-set header elements, <h1> to <h6> -->
+    <h1 id="header"><b>BEAT THAT😡🥊🤕🥲</b></h1>
+    <!-- The <div> tag defines a division or a section in an HTML document, and is commonly use as a "container" for nested HTML elements. -->
     <div id="container">
+      <p>
+        <b>
+          Hello! Welcome to BEAT THAT! Click submit to start the game.
+          Disclaimer: I know you just had a stressful day, but we are not
+          actually beating anyone.</b
+        >
+      </p>
```

Comment:
Nice attempt at changing the html and css! I died at this message LOL. I like that you give instructions to the user on how to start the game too. A very common comment I have for other projects is that when I enter the page, I just see a blank input field and output box and it is unclear how to proceed. Good that you're taking the user experience into account